app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventCard.kt

```diff
@@ -30,6 +35,12 @@ class EventCard(context: Context) : Card(CardManager.CARD_EVENT, context, "card_
     }
 
     override fun getNavigationDestination(): NavigationDestination? {
+        val event = this.event
+        if (event != null && event.kino != -1) {
+            val intent = Intent(context, KinoActivity::class.java)
+            intent.putExtra(Const.KINO_DATE, DateTimeUtils.getDateTimeString(event.startTime))
+            return SystemIntent(intent)
+        }
```

Comment:
A comment here would be nice, why we have two different details activities.... not a requirement tho.